configure.ac

```diff
@@ -151,6 +151,19 @@ AC_SUBST([git_commit_hash], "$GIT_COMMIT_HASH")
 AC_SUBST([DEV_VERSION_SUFFIX])
 AC_SUBST([dist_version], "$DIST_VERSION")
 
+AC_ARG_WITH([mono-framework],
+  AC_HELP_STRING([--with-mono-framework="MONO_FRAMEWORK"],
+  [Set where to look for the mono framework (macOS) [default=/Library/Frameworks/Mono.framework/Versions/Current]]),
+  with_mono_framework=$enableval,
+  with_mono_framework=/Library/Frameworks/Mono.framework/Versions/Current
```

Comment:
The whole feature is for macOS (although you _could_ use it elsewhere) so what other default should there be? I could put the whole thing inside an `AS_IF()` but ISTM that adding conditionals in the default value would make it harder to read.